channel.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-demo. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import (
+	"errors"
+
+	"perun.network/go-perun/backend/ethereum/wallet"
+	"perun.network/go-perun/channel"
+	"perun.network/go-perun/client"
+)
+
+type Allocation struct {
+	alloc *channel.Allocation
+}
+
+func (a *Allocation) Allocation() *channel.Allocation {
+	return a.alloc
+}
+
+type State struct {
+	state *channel.State
+}
+
+func (s *State) GetID() []byte {
+	return s.state.ID[:]
+}
+
+func (s *State) GetFirst() *BigInt {
+	return &BigInt{s.state.Allocation.Balances[0][0]}
+}
+
+func (s *State) GetSecond() *BigInt {
+	return &BigInt{s.state.Allocation.Balances[0][1]}
+}
+
+func NewAllocation(first, second *BigInt) *Allocation {
```

Comment:
Since we remove `Allocation`, we don't need this. But maybe a `NewBalances(first, second *BigInt) BigInts` would be handy.